### DISCUSSION TITLE
Run fsi scripts without dotnet SDK

### DISCUSSION THREAD

**Discussion by benwasd** [dotnet/fsharp#11927] [Q&A] [answered]
Is it possible to use F# interactive in net5.0 without the dotnet SDK? We use fsi on non-developer machines with only the runtime installed. 
I only see that F# interactive tools are published with VisualStudio or at https://www.nuget.org/packages/Microsoft.dotnet-interactive/. But dotnet tool install does not work without sdk and the visual studio installer only holds a net48 fsi.exe.

Did I miss something?
Thanks for any feedback.

**Comment by cartermp** [comment]
Not possible today and unlikely to ever be possible. I highly recommend that you don't use scripts in production but instead build a console app or service.

**Comment by baronfel** [comment]
If you did do such a thing, to achieve parity with the version that's packaged with the SDK you'd need to also make sure to at least include/package the Nuget #r handler dll alongside the binary (or manually reference it using the `--compilertool` argument).  Personally I think the tool itself should be more easily available, but there are a lot of implicit dependencies that a user of the non-packaged version would have to manully provide.

**Comment by KevinRansom** [comment, marked as answer]
@benwasd -- 
- fsi is an integral component of the dotnet sdk,  It has fairly deep dependencies on the sdk right now.
- We currently have no plans to break it out.

Thanks

Kevin
